.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Which editor is this for?

Reply:
Should be applicable for all editors that have the editorconfig extension installed